### ISSUE TITLE
Add documentation and samples for concurrency-limits

### ISSUE THREAD

**Issue by spencergibb** [spring-cloud/spring-cloud-netflix#3190]
https://github.com/Netflix/concurrency-limits

**Comment by spencergibb** [comment]
See https://medium.com/@NetflixTechBlog/performance-under-load-3e6fa9a60581

**Comment by eacdy** [comment]
When do we finish the doc?

**Comment by benhubert** [comment]
Do I get this right that this is the last task for supporting _Spring Boot 2.1_? How can I find out when _Spring Cloud Greenwich_ is going to be released?

**Comment by ryanjbaxter** [comment]
@benhubert https://github.com/spring-cloud/spring-cloud-release/milestones

**Comment by thekalinga** [comment]
@spencergibb @ryanjbaxter Will concurrency-limits be integrated with ribbon anytime soon?

**Comment by scalablepuppets** [comment]
I guess lack of documentation and any samples forcing me to look at netflix-concurrency dependencies and try/implement it. With no reference app and documentation its hard to follow a relationship b/w a RestController and actual service and how the concurrency limits will be configured at the controller layer.

**Comment by spencergibb** [comment]
This module has entered [maintenance mode](https://github.com/spring-cloud/spring-cloud-netflix#modules-in-maintenance-mode). This means that the Spring Cloud team will no longer be adding new features to the module. We will fix blocker bugs and security issues, and we will also consider and review small pull requests from the community.

**Comment by aruneinstein** [comment]
Why was this pushed out? What will be the preferred solution for concurrency in Spring Cloud since hystrix also has gone into maintenance?

**Comment by spencergibb** [comment]
h#ttps://github.com/resilience4j/resilience4j is the recommended replacement for hystrix
